src/scripts/main.js

```diff
@@ -1 +1,7 @@
-'use strict';
+import Menu from './components/menu';
+import Slider from './components/slider';
+import Form from './components/form';
```

Comment:
```suggestion
import './components/menu';
import './components/form';
import Slider from './components/slider';
```
Then, if you change export logic you can simple import file inside main)